xdsl/backend/riscv/traits.py

```diff
@@ -0,0 +1,21 @@
+from xdsl.ir import Operation
+from xdsl.traits import HasInsnsRepresentation
+
+
+class StaticInsnsRepresentation(HasInsnsRepresentation):
```

Comment:
```suggestion
class StaticInsnRepresentation(HasInsnRepresentation):
```
not really important but having it without the s should make the usage more consistent